src/utils/vm_image_vault_utils.cpp

```diff
@@ -80,7 +80,7 @@ QString mp::ImageVaultUtils::extract_file(const QString& file,
                                           bool delete_original) const
 {
     const auto fs_new_path = MP_FILEOPS.remove_extension(file.toStdU16String());
-    auto new_path = QString::fromStdString(fs_new_path.u8string());
+    auto new_path = QString::fromStdString(fs_new_path.string());
```

Comment:
This shouldn't make any difference on systems where the native encoding is already UTF-8 (e.g., Linux). But for Windows, the default is UTF-16, and the implementation actually re-encodes the string in narrow type (we should use wide string in Windows, but anyway). I've confirmed that there's no difference between C++17's u8string and string in Windows stdlib, so we should be safe on that front too.

https://github.com/microsoft/STL/blob/48db51b952de9a84b15a8b89b638a55e605235ef/stl/inc/filesystem#L932-L934

https://github.com/microsoft/STL/blob/48db51b952de9a84b15a8b89b638a55e605235ef/stl/inc/filesystem#L940-L947